README.md

```diff
@@ -35,8 +35,8 @@ LogicRs can be downloaded as a prebuilt package from the [releases](https://gith
 - `.AppImage` package (Linux)
 - `.zip` portable installation (Windows)
 
-Alternatively, you can download this repository and build LogicRs from source. 
-Please refer to [INSTALL.md](./INSTALL.md) for build instructions and information about compatability
+Alternatively, you can download this repository and build LogicRs from the source. 

```

Comment:
> build LogicRs from **the** source
Here, "source" refers to the source code, where it's common to say either "from source" or "from the source code".

Reply:
I see, I have made the appropriate changes :)

Reply:
Good, ty